### PR TITLE
Uploading new magnetic field by Mentink

### PR DESCRIPTION
**Pull request by jgalan**
This PR uploads the latest version of the BabyIAXO magnetic field to the repository.

### CODE REVIEW

File: magneticField/README.md
Comment (francandon) [inline magneticField/README.md:1]: I think there is a typo in the second definition, shouldn't be..
- 'Mentink_202401_cutoff.bin : ....'  ?

File: magneticField/README.md [hunk 1]
```diff
@@ -32,3 +32,7 @@ List of magnetic fields available
 - `Bykovskiy_202004.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X and Y, and 50mm in Z.
 
 - `Bykovskiy_202004.xls` : The original file generated with Matlab used to provide the file `Bykovskiy_202004.bin`.
+
+- `Mentink_202401.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -10m and 10m, reproducing the original file size.
```
Comment (jgalan) [inline magneticField/README.md:36]:
```suggestion
- `Mentink_202401.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -10m and 10m, reproducing the magnetic volume size as defined in the original file.
```

File: magneticField/README.md
Comment (jgalan) [reply magneticField/README.md:1]: Right!

File: magneticField/README.md [hunk 2]
```diff
@@ -33,6 +33,6 @@ List of magnetic fields available
 
 - `Bykovskiy_202004.xls` : The original file generated with Matlab used to provide the file `Bykovskiy_202004.bin`.
 
-- `Mentink_202401.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -10m and 10m, reproducing the original file size.
+- `Mentink_202401.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -10m and 10m, reproducing the magnetic volume size as defined in the original file.
 
 - `Mentink_202401.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -6m and 6m, introducing an artificial cut-off to the original file size. The field in that boundary is already below 0.1T.
```
Comment (francandon) [inline magneticField/README.md:38]:
```suggestion
- `Mentink_202401_cutoff.bin` : It contains the magnetic field definition for babyIAXO with a regular mesh precision of 10mm in X,Y and 50mm in Z. It defines the field between -6m and 6m, introducing an artificial cut-off to the original file size. The field in that boundary is already below 0.1T.
```

**Pull request merged**